src/Application/Actions/Hooks/StripeUpdate.php

```diff
@@ -112,4 +116,72 @@ public function handleChargeSucceeded(Event $event): Response
 
         return $this->respondWithData($event->data->object);
     }
+
+    public function handlePayoutPaid(Event $event)
+    {
+        $stripeClient = new \Stripe\StripeClient($this->apiKey);
```

Comment:
There is already a Stripe client available in the container, we should be making a new one.